swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    '''Child class that inherits from Item class (the parent class).'''
```

Comment:
Yay docstrings! Awesome inclusion of them here for and throughout your code for both classes and functions!